client/src/components/DataImportExport.vue

```diff
@@ -66,10 +66,16 @@ export default defineComponent({
     async function exportData() {
       exporting.value = true;
       try {
+        let response;
         if (isGlobal.value) {
-          await djangoRest.globalExport();
+          response = await djangoRest.globalExport();
         } else {
-          await djangoRest.projectExport(currentProject.value.id);
+          response = await djangoRest.projectExport(currentProject.value.id);
+        }
+        if (response.warnings) {
```

Comment:
Is response `null` here, or `{}`? I would have expected `null` but idk

Reply:
The backend sends a `Response` object with `detail` and `warnings`. I don't think there's a way to get a null response from this endpoint.

Reply:
It looks like it sends an empty 204 if there are no warnings. This `if` seems to be checking if there are any errors to display, implying that there is some sort of empty response indicating no errors.

Reply:
In that case, `response` here is an empty string because it is returned from `projectExport` as `(await ...).data`. The if statement doesn't result in any errors, it just evaluates to false. But for the sake of clarity, I changed the if statement in [96aac5f](https://github.com/OpenImaging/miqa/pull/423/commits/96aac5f99fa1b7442159c1e9bd7bf11ad3d3afbc)

Reply:
:+1: That should probably be included in the type definition